api/models/app.go

```diff
@@ -25,6 +27,7 @@ func (a *App) SetDefaults() {
 		// keeps the json from being nil
 		a.Config = map[string]string{}
 	}
+	a.ID = id.New().String()
```

Comment:
probably to be sure we should wrap with `if a.ID == ""` like everything else

Reply:
fixed